### DISCUSSION TITLE
colorbar plot

### DISCUSSION THREAD

**Discussion by kunal0kumar0sinha** [PMEAL/OpenPNM#2775] [Q&A]
I am trying to get color bar for this, but unable to do this. 
Code line- op.visualization.plot_connections(network=pn, color_by=q_throat, linewidth=3, ax=ax)
    _ = plt.axis('off')
Can you please help

![image](https://github.com/PMEAL/OpenPNM/assets/123363487/b1e7d9c5-8d47-40ab-a55a-830cebb16148)

**Comment by jgostick** [comment]
Try catching the object returned by the plot function in a variable, like ax, then doing plt.colorbar(ax).  I think that should work but not able to check right now.
